MaxText/configs/base.yml

```diff
@@ -63,15 +63,20 @@ int8_training: False
 # base_mlp_dim, base_num_decoder_layers and/or head_dim.
 global_parameter_scale: 1
 base_emb_dim: 2560
+base_num_heads: 8
 base_num_query_heads: 8
 base_num_kv_heads: 8
 base_mlp_dim: 8192
 base_num_decoder_layers: 16
 head_dim: 256
+
 # activation functions are .
 mlp_activations: ["relu"]
 dropout_rate: 0
 logits_via_embedding: True  # NOTE: this is True just for testing.
+norm_logits_via_embedding: True  # whether to normlize pre-softmax logits if logits_via_embedding is true
+logits_dot_in_fp32: True  # whether to use fp32 in logits_dense or shared_embedding dot product for stability
```

Comment:
There are already flags for this I believe? Might be wrong.

Reply:
Confirmed no existing flags in our repo. And they are needed for gpt3 implementation.